lib/lite_rolling_switch.dart

```diff
@@ -151,7 +165,9 @@ class _RollingSwitchState extends State<LiteRollingSwitch>
               ),
             ),
             Transform.translate(
-              offset: Offset(80 * value, 0),
+              offset: isRTL(context)
```

Comment:
_TODO: Apply the following correction to make code compatible with dynamic width component feature (dynamic calc on transform)._

```dart
Transform.translate(offset: isRTL(context)
	? Offset((-widget.width + 50) * value, 0) // -80
	:  Offset((widget.width - 50)  * value, 0) // 80
```